tests/test_fields.py

```diff
@@ -222,3 +222,20 @@ class TestNestedField:
     def test_nested_only_and_exclude_as_string(self, param):
         with pytest.raises(StringNotCollectionError):
             fields.Nested(Schema, **{param: 'foo'})
+
+    @pytest.mark.parametrize('unknown', (None, 'exclude', 'include', 'raise'))
+    def test_nested_unknown_override(self, unknown):
+        class NestedSchema(Schema):
+            class Meta:
+                unknown = 'exclude'
```

Comment:
Would it make sense to also parametrize the Meta attribute, so that all combinations are covered?

Current test would also pass if unknown defaulted to `EXCLUDE` instead of `RAISE`.

Also, in the rest of the tests, we use the capitalized global (`INCLUDE`) rather than its string value (`'include'`). Do you think you could change that?

Thanks.